docs/reference_guide.md

```diff
@@ -168,15 +168,21 @@ OPTIONS:
     --version      bpftrace version
 
 ENVIRONMENT:
-    BPFTRACE_STRLEN             [default: 64] bytes on BPF stack per str()
-    BPFTRACE_NO_CPP_DEMANGLE    [default: 0] disable C++ symbol demangling
-    BPFTRACE_MAP_KEYS_MAX       [default: 4096] max keys in a map
-    BPFTRACE_MAX_PROBES         [default: 512] max number of probes bpftrace can attach to
-    BPFTRACE_MAX_BPF_PROGS      [default: 512] max number of generated BPF programs
-    BPFTRACE_CACHE_USER_SYMBOLS [default: auto] enable user symbol cache
-    BPFTRACE_VMLINUX            [default: none] vmlinux path used for kernel symbol resolution
-    BPFTRACE_BTF                [default: none] BTF file
-    BPFTRACE_STACK_MODE         [default: bpftrace] Output format for ustack and kstack builtins
+    BPFTRACE_BTF                        [default: none] BTF file
+    BPFTRACE_CACHE_USER_SYMBOLS         [default: (see below)] enable user symbol cache
```

Comment:
The `see below` is a little misleading cuz this block is intended to be a copy paste of the binary output.

Reply:
I don't like it either but the default looks like this "Default: PER_PROGRAM if ASLR disabled or `-c` option given, PER_PID otherwise."

Reply:
Maybe "see docs" instead?

Reply:
Works for me.